internal/operators/ocs/consts.go

```diff
@@ -3,10 +3,14 @@ package ocs
 type ocsDeploymentMode string
 
 const (
-	ssdDrive     string            = "SSD"
-	hddDrive     string            = "HDD"
-	minDiskSize  int64             = 5 //5GB is the min disk size for OCS
-	compactMode  ocsDeploymentMode = "Compact"
-	minimalMode  ocsDeploymentMode = "Minimal"
-	standardMode ocsDeploymentMode = "Standard"
+	CPUCompactMode      int64             = 6  // per host requires 6 cpus for OCS in compact mode
+	CPUMinimalMode      int64             = 4  // per host requires at least 4 cpus for OCS in minimal mode
+	MemoryGBCompactMode int64             = 19 // per host requires at least 19 GB RAM for OCS in minimal mode
+	MemoryGBMinimalMode int64             = 11 // per host requires at least 11 GB RAM for OCS in minimal mode
```

Comment:
(11 * 3) +( 3 * 5) = 48 GB. The current requirement is 72 GB aggregate for all deployment modes, including minimal, for a single storage device set.

Reply:
@nbalacha  OCP requirements will also be added to OCS requirements, so for minimal mode it would be (11*3 + 3*5 + 8*3) = 72 GB,

Reply:
ack

Reply:
Please add that in a comment to make it clearer.

Reply:
@rewantsoni please add comments above all the CPU/Memory in different modes regarding the calculations.